flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/polardbx/PolardbxSourceTCase.java

```diff
@@ -0,0 +1,472 @@
+                                + "    tiny1_c BOOLEAN,\n"
+                                + "    boolean_c BOOLEAN,\n"
+                                + "    date_c DATE,\n"
+                                + "    time_c TIME(0),\n"
+                                + "    datetime3_c TIMESTAMP(3),\n"
+                                + "    datetime6_c TIMESTAMP(6),\n"
+                                + "    timestamp_c TIMESTAMP(0),\n"
+                                + "    file_uuid BYTES,\n"
+                                + "    bit_c BINARY(8),\n"
+                                + "    text_c STRING,\n"
+                                + "    tiny_blob_c BYTES,\n"
+                                + "    blob_c BYTES,\n"
+                                + "    medium_blob_c BYTES,\n"
+                                + "    long_blob_c BYTES,\n"
+                                + "    year_c INT,\n"
+                                + "    enum_c STRING,\n"
+                                + "    set_c ARRAY<STRING>,\n"
+                                + "    json_c STRING,\n"
+                                + "    point_c STRING,\n"
+                                + "    geometry_c STRING,\n"
+                                + "    linestring_c STRING,\n"
+                                + "    polygon_c STRING,\n"
+                                + "    multipoint_c STRING,\n"
+                                + "    multiline_c STRING,\n"
+                                + "    multipolygon_c STRING,\n"
+                                + "    geometrycollection_c STRING,\n"
+                                + "    primary key (`id`) not enforced"
+                                + ") WITH ("
+                                + " 'connector' = 'mysql-cdc',"
+                                + " 'scan.incremental.snapshot.enabled' = 'true',"
+                                + " 'hostname' = '%s',"
+                                + " 'port' = '%s',"
+                                + " 'username' = '%s',"
+                                + " 'password' = '%s',"
+                                + " 'database-name' = '%s',"
+                                + " 'table-name' = '%s',"
+                                + " 'scan.incremental.snapshot.chunk.size' = '100',"
+                                + " 'server-id' = '%s'"
+                                + ")",
+                        HOST_NAME,
+                        PORT,
+                        USER_NAME,
+                        PASSWORD,
+                        DATABASE,
+                        getTableNameRegex(captureCustomerTables),
+                        getServerId());
+        tEnv.executeSql(sourceDDL);
+
+        TableResult tableResult = tEnv.executeSql("select * from polardbx_full_types");
+        CloseableIterator<Row> iterator = tableResult.collect();
+        List<String> realSnapshotData = fetchRows(iterator, 1);
+        assertEquals(1, realSnapshotData.size());
```

Comment:
let's compare the result instead of the size of result

Reply:
thanks and done